app/src/main/java/org/xbmc/kore/provider/MediaDatabase.java

```diff
@@ -346,6 +354,7 @@ public void onCreate(SQLiteDatabase db) {
                    MediaContract.SyncColumns.UPDATED + " INTEGER NOT NULL," +
                    MediaContract.SongsColumns.HOST_ID + " INTEGER NOT NULL " + References.HOST_ID + ", " +
                    MediaContract.SongsColumns.ALBUMID + " INTEGER NOT NULL, " +
+                   MediaContract.SongsColumns.ARTISTID + " INTEGER NOT NULL, " +
```

Comment:
Maybe i'm missing something, but you're creating an Integer column, and them you're updating it with a string, which is a concatenation of the artistid's for the song. If there's more than one, this will probably crash.